examples/data-objects/key-value-cache/src/index.ts

```diff
@@ -101,11 +101,12 @@ class KeyValue implements IKeyValue, IFluidObject, IFluidRouter {
     }
 }
 
-export class KeyValueFactoryComponent implements IRuntimeFactory, IFluidDataStoreFactory {
+export class KeyValueFactoryComponent
+    extends RuntimeFactoryHelper
+    implements IFluidDataStoreFactory {
```

Comment:
Nit: I find it hard to read when there is indentation  does not change after {. Consider this style:
```typescript
export class KeyValueFactoryComponent
    extends RuntimeFactoryHelper
    implements IFluidDataStoreFactory
{
    public static readonly type = "@fluid-example/key-value-cache";
```